.github/workflows/new-label.yml

```diff
@@ -12,49 +12,69 @@ on: # any labels added to an issue
 
     - name: log action output
-      run: echo ${{steps.run_action.outputs.message}}     
+      run: echo ${{steps.run_action.outputs.message}}
   new-issue:
+    if: ${{ github.event.label.name == 'investigating' }}
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2.0.0 
+    - uses: actions/checkout@v2.0.0
     - name: issue comment
       id: issue_comment
       uses: ./
-      with: 
+      with:
         repo-token: ${{ secrets.GITHUB_TOKEN}}
         label: 'investigating'
         message: >
-          Thank you for filing this issue.  We appreciate your feedback and will review the issue as soon as possible. 
-          Remember, however, that GitHub isn't a mechanism for receiving support under any agreement or SLA. 
+          Thank you for filing this issue.  We appreciate your feedback and will review the issue as soon as possible.
+          Remember, however, that GitHub isn't a mechanism for receiving support under any agreement or SLA.
           If you require immediate assistance, contact Salesforce Customer Support.
     - name: log action output
-      run: echo ${{steps.run_action.outputs.message}}  
+      run: echo ${{steps.run_action.outputs.message}}
   new-feature:
+    if: ${{ github.event.label.name == 'feature' && !contains(github.event.issue.labels.*.name, 'owned by another team') }}
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2.0.0 
+    - uses: actions/checkout@v2.0.0
     - name: issue comment
       id: issue_comment
       uses: ./
-      with: 
+      with:
         repo-token: ${{ secrets.GITHUB_TOKEN}}
         label: 'feature'
         message: >
           Thank you for filing this feature request. We appreciate your feedback and will review the feature at our next grooming or sprint planning session. We prioritize feature requests with more upvotes and comments.
     - name: log action output
-      run: echo ${{steps.run_action.outputs.message}}  
+      run: echo ${{steps.run_action.outputs.message}}
+  new-feature-on-another-team:
+    if: >-
+      (github.event.label.name == 'feature' || github.event.label.name == 'owned by another team')
+      &&
+      (contains(github.event.issue.labels.*.name, 'feature') && contains(github.event.issue.labels.*.name, 'owned by another team'))
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2.0.0
+    - name: issue comment
+      id: issue_comment
+      uses: ./
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN}}
+        message: >
+          Thank you for filing this feature request. However, we have determined that the functionality you've requested would need to be completed by another team. Please submit your request to the [Salesforce IdeaExchange](https://trailblazer.salesforce.com/ideaSearch). Once submitted, please post a link to it in this issue so that other can upvote your idea.
```

Comment:
```suggestion
          Thank you for filing this feature request. However, we've determined that the functionality you've requested must be completed by another team. Submit your request to the [Salesforce IdeaExchange](https://trailblazer.salesforce.com/ideaSearch). Then post a link to the request in this issue so that others can upvote your idea.
```